app/routes/index.md

```diff
@@ -3,4 +3,47 @@ meta:
+
+The new GCN is the next generation transient alert system built upon the legacy of the Gamma-ray Coordinates Network (referred to throughout as GCN Classic, formerly BACODINE/TAN), updated to use the modern technologies for transient distribution being utilized by the optical transient community and providing a robust reliable cloud-based system.
+
+## The GCN system distributes:
+
+- **Notices:** Locations of GRBs and other Transients (e.g. gravitational waves, neutrinos, AGN flares) detected by space- and ground-based observatories (most in realtime while the source is still bursting and others are that delayed due to communications and processing).
+- **Circulars:** Reports of follow-up observations made by ground-based and space-based optical, radio, X-ray, gamma-ray, TeV, and other particle observers.
+
+These two functions provide a one-stop shopping network for follow-up sites and GRB and transient researchers. The new GCN system uses the [Apache Kafka](https://kafka.apache.org) protocol for distributing Notices, and is operated within the Commercial managed Kafka system, [Confluent Platform](https://www.confluent.io), running on Amazon Web Services. The Kafka systems utilizes Hopclient, developed by SCiMMA, and ...
+
+## The GCN system has three ways to access the data
+
+| System                 | Transmission Protocols | Data Formats                   |
+| ---------------------- | ---------------------- | ------------------------------ |
+| GCN Classic            | Email, Socket          | Text, 160 byte Binary, VOevent |
+| GCN-Classic-over-Kafka | Kafka                  | Text, 160 byte Binary, VOevent |
+| GCN                    | Kafka                  | AVRO/JSON                      |
+
+### GCN Classic
+
+Legacy system run on premises at NASA/GSFC utilizing bespoke protocols for ingestion and distribution of transient data. New web application to modify your configurations coming soon. The GCN Classic system is being maintained until the community has fully transitioned to the new GCN
+
+### GCN-Classic-over-Kafka
+
+Transition system serving legacy GCN Classic formats via Kafka.
+
+### GCN
+
+New system serving data in AVRO schema, which are JSON records. These schema have been formatted to provide uniformity between instruments wherever possible making it easier to compare records across missions. GCN provides both instrument specific topics as well as combined event based topics.
+
+## Other Components of the New GCN
+
+### GRB Name Server
+
+In parnership with the [Transient Name Server](https://www.wis-tns.org), the GCN team is building a GRB name server that will allow GRB-detecting instruments to both name GRBs and retreive names of already discovered GRBs. These names will be automatically ingested into GCN and associated with GCN notices and circular for the same event. This feature will be available soon.
```

Comment:
I'm hesitant to list this on the front page because we don't really have any idea what "soon" means in this case. I think we should avoid describing things that are "under construction".